server/metrics/metrics.go

```diff
@@ -137,6 +140,24 @@ func NewMetrics(info InstanceInfo) *Metrics {
 	})
 	m.registry.MustRegister(m.linkedChannelsTotal)
 
+	m.changeEventQueueCapacity = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace:   MetricsNamespace,
+		Subsystem:   MetricsSubsystemApp,
```

Comment:
Why one is `MetricsSubsystemApp` and the other is `MetricsSubsystemAPI`, shouldn't be both `App` or both `API`?

Reply:
Good catch! Fixed :)